tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -92,12 +92,20 @@ def check_args(**kwargs):
     require_arg(kwargs, "binary")
 
 
-def browser_kwargs(logger, test_type, run_info_data, config, **kwargs):
+def browser_kwargs(logger, test_type, run_info_data, config, subsuite, **kwargs):
+    binary_args = kwargs["binary_args"][:]
+    if subsuite.config.get("binary_args"):
```

Comment:
We need check subsuite if not '' first, otherwise subsuite.config is invalid. Same for line 101.